Sming/Makefile-rboot.mk

```diff
@@ -248,11 +253,6 @@ ifeq ($(ENABLE_SSL),1)
 	CXXFLAGS += $(AXTLS_FLAGS)	
 endif
 
-ifeq ($(ENABLE_CUSTOM_LWIP), 1)
```

Comment:
Because `LWIP_INCDIR` is already being added to `EXTRA_INCDIR` on line 180 of my `Makefile-rboot.mk` (or line 174 of yours). While apparently the path is not the same (it doesn't end in `/include`), if you look at `Makefile-project.mk`, you'll see that these lines are not there, either, and only the path that doesn't end in `include` is ever added to `EXTRA_INCDIR` (or is it wrong too? doubt so, as that would have broken custom LWIP for everyone, I think).

Furthermore, the path it was adding to `EXTRA_INCDIR` was missing the `$(SMING_HOME)` prefix and I doubt it would work when the makefile is run from the project directory.